python/mixedDetector_photometry.py

```diff
@@ -0,0 +1,171 @@
+import os
+import numpy as np
+import matplotlib.pyplot as plt
+from lsst.sims.photUtils import Bandpass, Sed, PhotometricParameters, LSSTdefaults, SignalToNoise
+import bandpassUtils as bu
+import sedUtils as su
+
+def calcM5s(hardware, system, title='m5'):
+    photParams = PhotometricParameters()
+    lsstDefaults = LSSTdefaults()
+    darksky = Sed()
+    darksky.readSED_flambda(os.path.join('../siteProperties', 'darksky.dat'))
```

Comment:
This should also use lsst.utils.getPackageDir so that it does not depend on where it is being run from